metadata-ingestion/src/datahub/ingestion/source/metabase.py

```diff
@@ -60,6 +60,14 @@ class MetabaseConfig(DatasetLineageProviderConfigBase):
         default=None,
         description="Custom mappings between metabase database engines and DataHub platforms",
     )
+    platform_instance_map: Optional[Dict[str, str]] = Field(
```

Comment:
We don't need this platform_instance_map here. Its already present in superclass `DatasetLineageProviderConfigBase`

Reply:
Thank you for noticing that. I've removed this definition and rebased changes onto fresh `master`. All related tests seem to be fine.